webp-to-png@tomunek/webp-to-png@tomunek.nemo_action.in

```diff
@@ -0,0 +1,8 @@
+[Nemo Action]
+_Name=Convert to PNG
+_Comment=Convert selected WEBP file to PNG
+Exec=<webp-to-png@tomunek/webp-to-png@tomunek.sh %F>
+Icon-Name=document-properties-symbolic
+Selection=s
```

Comment:
You could theoretically expand this to cover multiple files at some point if you want.

Reply:
I am considering adding support for WEBM (to MP4).
Is it allowed to change the name of the extension (for example to webconv) if I decide to expand it's functionality?

Reply:
Of course! The action name itself is the only thing that is stuck but the functionality can be expanded as you would like.